cmd/pj-rehearse/main.go

```diff
@@ -1,23 +1,278 @@
 package main
 
 import (
+	"encoding/json"
 	"flag"
 	"fmt"
-	"log"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	// TODO: Solve this properly
```

Comment:
What is this and have we solved it?

Reply:
Importing a lib for deepcopy felt like a temporary workaround. It's not because prowconfig.Presubmit & friends do not have DeepCopy themselves.